go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/ava-labs/awm-relayer
 go 1.20
 
 require (
-	github.com/ava-labs/avalanchego v1.10.17
-	github.com/ava-labs/coreth v0.12.9-rc.9
-	github.com/ava-labs/subnet-evm v0.5.10
-	github.com/ava-labs/teleporter v0.0.0-20240108172200-f03f526e5312
+	github.com/ava-labs/avalanchego v1.10.18
+	github.com/ava-labs/coreth v0.12.10-rc.5
+	github.com/ava-labs/subnet-evm v0.5.11
+	github.com/ava-labs/teleporter v0.1.1-0.20240125232816-d92ef8e89b7a
```

Comment:
I have so many questions about this tag... How did we get `v0.1.1-0`? I don't see that tag in the teleporter repo. Also why is there a dash? Was this supposed to be an rc?

Reply:
This includes the changes in https://github.com/ava-labs/teleporter/pull/256. `d92ef8e89b7a` is the first commit of that PR - I'll go back and fetch the latest commit once that PR is merged.

From what I've observed, when fetching a non-tagged commit using `go get`, go.mod will take the latest semver release and automatically increment the bugfix version before appending the timestamp and commit hash.

The changes to go.mod here were made by running `go get github.com/ava-labs/teleporter@d92ef8e89b7a`

Reply:
Okay thank you!